src/terminal/adapter/adaptDispatchGraphics.cpp

```diff
@@ -50,17 +50,12 @@ void AdaptDispatch::s_ApplyColors(WORD& attr, const WORD applyThis, const bool i
     // Mask off only the foreground or background
     if (isForeground)
     {
-        attr &= ~(FOREGROUND_BLUE | FOREGROUND_GREEN | FOREGROUND_RED | FOREGROUND_INTENSITY);
-        wNewColors &= (FOREGROUND_BLUE | FOREGROUND_GREEN | FOREGROUND_RED | FOREGROUND_INTENSITY);
+        WI_UpdateFlagsInMask(attr, FOREGROUND_BLUE | FOREGROUND_GREEN | FOREGROUND_RED | FOREGROUND_INTENSITY, wNewColors);

```

Comment:
oh, i love this

Reply:
i feel like we may actually have a constant laying around for that mask. If not, should we?

Reply:
it turns out we do: `FG_ATTRS` and `BG_ATTRS` and `META_ATTRS`

It's a define and not a constant, though, but that should be fine.

Reply:
Fixed. Now it's finally easy on the eyes.